_episodes/15-coding-conventions.md

```diff
@@ -313,39 +313,38 @@ use the other one to avoid backslashes in the string as it improves readability.
 ### Naming Conventions
 There are a lot of different naming styles in use, including:
 
-- b (single lowercase letter)
-- B (single uppercase letter)
-- lowercase
-- lower_case_with_underscores
-- UPPERCASE
+- lower_case_with_underscores (or snake_case)
 - UPPER_CASE_WITH_UNDERSCORES
 - CapitalisedWords (or PascalCase) (note: when using acronyms in CapitalisedWords, capitalise all the letters of the acronym,
 e.g HTTPServerError)
 - camelCase (differs from CapitalisedWords/PascalCase by the initial lowercase character)
 - Capitalised_Words_With_Underscores
 
 As with other style guide recommendations - consistency is key.
-Pick one and stick to it, or follow the one already established if joining a project mid-way.
+Follow the one already established in the project, if there is one.
+If there isn't, follow any standard language style (such as
+[PEP8](https://www.python.org/dev/peps/pep-0008/) for Python).
+Failing that, just pick one, document it and stick to it.
+
 Some things to be wary of when naming things in the code:
 
-- Avoid using the characters 'l' (lowercase letter L),
-  'O' (uppercase letter o),
-  or 'I' (uppercase letter i)
-  as single character variable names.
-  In some fonts, these characters are indistinguishable from the numerals one and zero.
-  When tempted to use 'l', use 'L' instead.
-- Avoid using non-ASCII (e.g. UNICODE) characters for identifiers
+- Avoid any names that could cause confusion (e.g. lower case `l` is
+  hard to distinguish from a `1` (one)).
```

Comment:
```suggestion
  hard to distinguish from a `1` (one), 'O' (uppercase o) from a '0' (zero),
 'I' (uppercase i) from 'l' (lowercase L)).
```

Reply:
I'd still keep the other examples that may cause confusion as people may not be fully aware.